support/insights-operator/about-insights-operator.adoc

```diff
@@ -0,0 +1,17 @@
+[id="about-insights-operator"]
+= About Insight Operator
+include::modules/common-attributes.adoc[]
+:context: insights-operator
+
+toc::[]
+
+Insights-operator collects anonymized information about:
+
+* The version of the {product-title} cluster and it's components
+* The health, status and current errors reported by {product-title} components and nodes
+* Network topology information
+* The features in use
```

Comment:
This probably needs to be re-worded. Something like: 

- Provides information if a product Feature is in use
    - Example: Service Mesh, Serverless (knative), etc.